res/code_templates/field/varchar/custom/Extension/modules/__module__/Ext/Vardefs/sugarfield___field__.php.twig

```diff
@@ -17,12 +17,12 @@ $dictionary["{{ moduleBean }}"]["fields"]["{{ field }}"] = array(
     'comment' => '',
     'help' => '',
     // Sugar search features
-    'unified_search' => true,
-    'full_text_search' => array(
-        'enabled' => true,
-        'searchable' => true,
-        'boost' => 1.00
-    ),
+    //'unified_search' => false,
+    //'full_text_search' => array(
+    //    'enabled' => false,
+    //    'searchable' => true,
```

Comment:
It was already there for search feature enabled fields. If we do not want to enable search feature we can remove these lines altogether. That is why I commented them.